### ISSUE TITLE
issue with rsnapreport.pl when I upgraded to 1.4.2-2

### ISSUE THREAD

**Issue by Philippe34** [rsnapshot/rsnapshot#172]
Hi,
I have now issues with rsnapreport.pl when I upgraded to new rsnapshot 1.4.2-2.el6 version.
I used before rsnapreport.pl of rsnapshot 1.3.1-12 in centOS6 and I had correct operating:

```
SOURCE                          TOTAL FILES   FILES TRANS      TOTAL MB     MB TRANS   LIST GEN TIME  FILE XFER TIME
/home                         	    1493418            34    1060692.93      4355.50   0.001 seconds   0.000 seconds
```

I upgraded with epel.repo to rsnapshot 1.4.2-2 and now I have this issue:
```
ERRORS
/home/ NO STATS DATA
```

I installed also rsnapshot 1.4.2-2.el7 on an another centOS7 machine, and it is the same problem
```
ERRORS
/encryptedfs/owncloud/ NO STATS DATA
```

My configuration did not change, I have in rsnapshot.conf:
```
cmd_cp          /bin/cp
rsync_long_args --delete --numeric-ids --relative --delete-excluded --stats
verbose         3
```

My crontab is:
```
01      */6     *       *       *       /usr/bin/rsnapshot hourly 2>&1 | /usr/local/bin/rsnapreport.pl | mail -s "rsnapshot hourly ponant" admin@ies.univ-montp2.fr
```

And no errors in /var/log/rsapshot:
```
/usr/bin/rsnapshot hourly: started
echo 25674 > /var/run/rsnapshot.pid
/usr/local/sbin/preexec_sync_backup
/bin/rm -rf /people/backup/snapshot/hourly.3/
mv /people/backup/snapshot/hourly.2/ /people/backup/snapshot/hourly.3/
mv /people/backup/snapshot/hourly.1/ /people/backup/snapshot/hourly.2/
/bin/cp -al /people/backup/snapshot/hourly.0 /people/backup/snapshot/hourly.1
/usr/bin/rsync -a --delete --numeric-ids --relative --delete-excluded --stats --exclude='/home/lost+found' --exclude='/home/aquota.user' --exclude='perso/' --exclude='.cache/' --exclude='*.iso' /home /people/backup/snapshot/hourly.0/localhost/
touch /people/backup/snapshot/hourly.0/
/usr/local/sbin/create_backupDir
rm -f /var/run/rsnapshot.pid
/usr/bin/rsnapshot hourly: completed successfully
```

I can not find solutions to this issue.
What should I do ?
Thanks for your help.

**Comment by bebehei** [comment]
Probably [in the changes between 1.3.1 and 1.4.2](https://github.com/rsnapshot/rsnapshot/compare/1.3.1...master), no one actually tested `rsnapreport.pl` to work properly with rsnapshot.

Yes, this is a bug, but I'm not eager to research the problem and fix it. I haven't even used `rsnapreport.pl` ever. Sorry.

Most fortunate solution would be just downgrading to 1.3.1 again. Sorry.

**Comment by Philippe34** [comment]
Thank you for your sincere response.
rsnapreport.pl is not essential for the functioning of rsnapshot and backups. I'm not going to downgrading to have this functionality, but it is a practical tool to have an idea of the volumetry of our data. 
However I have found the solution. It is not really a bug, but I had to increase the verbosity level for it works.
Just put extra verbose in rsnapshot.conf:
`verbose         4`

instead of : verbose    3

It would be nice to update the documentation in rsnapreport.pl for the next release.

**Comment by bebehei** [comment]
Ironically I found the fix... 79e25dc83bc234357cf3dbbfa6694e61abd30e6b 🤦‍

You're using `/usr/local/bin/rsnapreport.pl` (**local**) are you sure, this is the correct `rsnapreport.pl`?

I don't know how rsnapshot is shipped in your distro (with or without rsnapreport?), but it may be also a good choice to maybe get the `rsnapreport.pl` from master.

**Comment by Philippe34** [comment]
My rsnapreport.pl is shipped in the package: rsnapshot-1.4.2-2.el7.noarch
```
# rpm -ql rsnapshot | grep rsnapreport.pl
/usr/share/doc/rsnapshot-1.4.2/utils/rsnapreport.pl
```
Just for convenience, I copied ../utils/rsnapreport.pl into /usr/local/bin/ and it is very good with "verbose 4"

I can not use rsnapreport.pl from master since I have rsync-3.0.9-17.el7.x86_64.
In master, it works with rsync >= 3.1.0

Thanks
